Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Private/InteropCodeGenerator/SpatialGDKEditorInteropCodeGenerator.cpp

```diff
@@ -128,10 +131,22 @@ FString GenerateIntermediateDirectory()
 bool SpatialGDKGenerateInteropCode()
 {
 	const USpatialGDKEditorToolbarSettings* SpatialGDKToolbarSettings = GetDefault<USpatialGDKEditorToolbarSettings>();
-	ClassHeaderMap InteropGeneratedClasses;
-	for (auto& Element : SpatialGDKToolbarSettings->InteropCodegenClasses)
+	TArray<UClass*> InteropGeneratedClasses;
+	for (TObjectIterator<UClass> It; It; ++It)
 	{
-		InteropGeneratedClasses.Add(Element.ReplicatedClass, Element.IncludeList);
+		if (It->HasAnySpatialClassFlags(SPATIALCLASS_GenerateTypeBindings) == false)
+		{
+			continue;
+		}
+
+		// Ensure we don't process skeleton or reinitialised classes
+		if (	It->GetName().StartsWith(TEXT("SKEL_"), ESearchCase::CaseSensitive)
+			||	It->GetName().StartsWith(TEXT("REINST_"), ESearchCase::CaseSensitive))
```

Comment:
Feel like the spacing is weird here.

Reply:
I think it's Git just being weird. Alignment looks fine on mine.